articles/blockchain-as-a-revolution-inhealth/index.md

```diff
@@ -0,0 +1,86 @@
+
+***Security:*** Transactions on the blockchain network must first be verified by other blockchain users before they are recorded. Once verified, the transactions are encrypted and linked to a previous transaction using cryptography. [The blockchain is virtually impossible to hack the because  data is stored across a network of devices](https://medium.com/pikciochain/how-is-blockchain-revolutionizing-healthcare-7f6d2a48e561).
+
+***Transparency:*** Before an existing version is changed, everyone in the blockchain network must agree since the records are shared. An overhaul of the whole network would be required to change a single transaction on a blockchain network, making the data on the network transparent, consistent and largely accurate. All participants can access any transaction on the blockchain as long as they have a private key, as every transaction is decentralized. This makes it easier to share information across multiple platforms.
+
+### Implementing Blockchain in health care
+![Health care img](/engineering-education/blockchain-as-a-revolution-inhealthcare/Healthcare-img.jpg)
+[*Image Source: https://www.gettyimages.com/detail/photo/nurse-and-surgeon-inspecting-screens-in-operating-royalty-free-image/1202320470*](https://www.gettyimages.com/detail/photo/nurse-and-surgeon-inspecting-screens-in-operating-royalty-free-image/1202320470)
+
+There are numerous applications of Blockchain in healthcare:
+
+### Record Keeping
+***Medical History Tracking***
+With Blockchain, patients' treatment history can be tracked anywhere and at any time with no ambiguity, hence providing health practitioners a better context.
+
+_In Data Security: [Do you know that your medical data is worth more than your credit card number in the black market?](https://medium.com/pikciochain/how-is-blockchain-revolutionizing-healthcare-7f6d2a48e561) According to a report by Protenus Breach Barometer, about 140 million patient health records were hacked between 2015 and 2016._
+>**Blockchain has the potential to keep private information safe and secure while still being accessible anywhere.**
+
+***Medical Device Usage Tracking***
+The use of  medical devices such as electronic wearables have been on the rise, with new reports revealing that forty percent of medical patients use such devices to track their health. However, this makes it difficult to store all the  daily medical reports from devicse for each patients. This is another area where keeping blockchain records can help.
```

Comment:
Note that formatting is flexible but must be consistent throughout the article. A previous version had 40% percent. You can write "forty percent" or "40%," but keep the same format consistent.

Reply:
Expand on how blockchain records can help track individual patient device usage in addition to hospital or institutional information.

Reply:
Done